### PR TITLE
Add ros2_sources to ros2-development role

### PR DESCRIPTION
**Pull request by emersonknapp**
It seems like in general we probably want the sources for the dev role?

### CODE REVIEW

**Comment by prajakta-gokhale** [comment]
@nuclearsandwich please take a look!

File: roles/ros2-development.json [hunk 1]
```diff
@@ -16,5 +16,5 @@
     }
   },
   "chef_type": "role",
-  "run_list": [ "recipe[ros2_windows::ros2]" ]
+  "run_list": [ "recipe[ros2_windows::ros2]", "recipe[ros2_windows::ros2_sources]" ]
```
Comment (brawner) [inline roles/ros2-development.json]: Could you put these on separate lines for better readability?

**Review by brawner — COMMENTED** [review]
Looks good to me, just comment about style

**Review by brawner — APPROVED** [review]

**Comment by brawner** [comment]
Thanks for the patch. This PR was needed because originally I had just intended to use the download_sources parameter in an if statement in the ros2.rb recipe. But we diverted from that style and put things in run lists instead. However, since this one wasn't part of the CI work, it was forgotten about.

**Comment by emersonknapp** [comment]
Sounds good, thanks for merging!

**Pull request merged**